nncf/experimental/openvino_native/graph/nncf_graph_builder.py

```diff
@@ -54,6 +57,35 @@ def convert_ov_dtype_to_nncf_dtype(ov_dtype: str) -> Dtype:
             raise NotImplementedError(f'NNCF is not yet supported OpenVINO data type: {ov_dtype}.')
         return Dtype(conversion_map[ov_dtype])
 
+    @staticmethod
+    def _filter_weight_input_ports(inputs: List[ov.Input], metatype: Type[OperatorMetatype]) -> List[ov.Input]:
+        """
+        Specifies the possible weight ports of the OpenVINO node.
+
+        :param inputs: OpenVINO node inputs.
+        :param metatype: NNCF meta type which corresponds to operation.
+        :return: OpenVINO node inputs that may contain weights.
+        """
+        if metatype == OVConvolutionBackpropDataMetatype:
+            return inputs[:2]
+        return inputs
+
+    @staticmethod
+    def _add_nncf_node(node: ov.Node, graph: NNCFGraph) -> None:
+        """
+        Creates NNCFNode from OpenVINO node and adds to the NNCFGraph.
+
+        :param node: OpenVINO node.
+        :param graph: NNCFGraph.
+        """
+        ov_dtype = node.get_element_type().get_type_name()
+        nncf_dtype = GraphConverter.convert_ov_dtype_to_nncf_dtype(ov_dtype)
+        node_type = node.get_type_name()
+        metatype = OV_OPERATION_METATYPES.get_operator_metatype_by_op_name(node_type)
```

Comment:
Some discrepancy between the naming `operation` and `operator`.

Reply:
Done